File: Arcade/Intro/C++/29_chessBoardCellColor.cpp
bool chessBoardCellColor(std::string cell1, std::string cell2) {
    int onex=cell1[0]-'A'+1;
    int oney=cell1[1]-'0';
    bool oneD=onex%2==oney%2;
    
    int twox=cell2[0]-'A'+1;
    int twoy=cell2[1]-'0';
    bool twoD=twox%2==twoy%2;
    
    return oneD==twoD;
}

File: Arcade/Intro/JavaScript/29_chessBoardCellColor.js
function chessBoardCellColor(cell1, cell2) {
    var dcol = cell1.charCodeAt(0)-cell2.charCodeAt(0);
    var drow = cell1.charCodeAt(1)-cell2.charCodeAt(1);
    return (dcol + drow) % 2 !== 0 ? false : true;
}

